.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+| Question         | Answer
+| ---------------- | ---
+| Branch?          | `master` for hotfix / `develop` for bug fixes <!-- see below -->
+| Bug fix?         | yes/no
+| New feature?     | yes/no
+| Breaking change? | yes/no
```

Comment:
Maybe we could have more details 😄 

I think about `Potential side-effects`

Reply:
I use the `Breaking change` to speak the same as our Git commit convention.

https://github.com/angular/angular/blob/master/CONTRIBUTING.md#footer

Reply:
I mean, maybe we can encourage having more details than mere `yes/no` and say

```
There will be breaking changes at those locations:
- location/breaking change
- location/breaking change
```

Reply:
I've added a `## Description` section